modules/core/shared/src/main/scala/lucuma/core/syntax/effect.scala

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2016-2025 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.syntax
+
+import cats.FlatMap
+import cats.Functor
+import cats.data.NonEmptyChain
+import cats.effect.Outcome
+import cats.effect.Spawn
+import cats.syntax.all.*
+
+trait effect:
+  extension [F[_]](F: Spawn[F])
+    /**
+     * Similar to `F.race`, but in case of error in the winner fiber, keeps the other one. Note that
```

Comment:
shoould we use this for the simbad query?

Reply:
Yes, I'm planning to use it in explore when train reaches there.